app/models/queue_item.rb

```diff
@@ -1,22 +1,38 @@
 class QueueItem < ActiveRecord::Base
+  default_scope { order(:position) }
```

Comment:
Be careful with `default_scope`, it can have unintended consequences. You are affecting every place in your entire application which uses `QueueItem`. It is usually recommended to create named scopes instead.